tools/ctl/ctl.c

```diff
@@ -114,29 +146,29 @@ static void header_dump(struct sof_abi_hdr *hdr)
 }
 
 /* dump binary data out with 16bit hex format */
-static void binary_data_dump(unsigned int *user_data, int raw_data)
+static void binary_data_dump(void)
```

Comment:
Pass the priv data struct.

Reply:
where do we write the binary data to file ?

Reply:
We didn't write the binary data to file.

But as you suggested that further refinements are required, let me find some time to figure it out like that.

Reply:
output to file feature added, addressed.